go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/hudl/fargo
+
+go 1.16
```

Comment:
If you are upgrading go please upgrade Dockerfile in /docker/ directory as well

Reply:
Done